github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -385,7 +385,7 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
-  - always_run: false
+  - always_run: true
```

Comment:
Can we make it always run but optional for now? Just for this week to see how it behaves?

Reply:
sure. restored `optional: true`